qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -93,6 +118,26 @@ def __init__(self, control_config: Dict[Tuple[int, ...], List[ControlChannel]] =
         self._hash_to_counter_map = {}
         self._parameter_counter = 0
 
+        self._library = None
+        if library is not None:
+            self._library = library
+
+            # Add the basis gates
+            for gate in library.basis_gates:
+                self.add_schedule(library[gate], num_qubits=library.num_qubits(gate))
+
+            # Add the default values
+            if add_parameter_defaults:
```

Comment:
When one builds from a serialized instance which contains parameter values we do not want to add the default values because otherwise we will add them twice: once from the library and once from the serialized data.

Reply:
Okay, that is fine. I don't like having a argument that the user never needs to use but I can't think of a better way right now. `from_config` could set things up carefully somehow (like not pass a library to `__init__` and then set the library separately) but I can't think of a way to do that that is not brittle.